src/modules/search/components/ResultsFilter.jsx

```diff
@@ -0,0 +1,74 @@
+import * as React from 'react';
+import styled from 'styled-components';
+const scIcon = require('./images/soundcloud.svg');
+const ytIcon = require('./images/youtube.svg');
+
+export default function ResultsFilter(props) {
+	const choices = [
+		{name: 'YoutubeResult', icon: ytIcon},
+		{name: 'SoundCloudResult', icon: scIcon},
+		{name: 'both'}
+	];
+	return (
+		<Filter>
+			{choices.map(choice => (
+				<Option key={choice.name}>
+					<StyledRadio
+						type="radio"
+						name="filter"
+						onClick={props.handleFilter}
+						id={choice.name}
+						defaultChecked={choice.name === 'both' ? true : false}
```

Comment:
Similarly, any time you have a ternary where either branch is a constant `true` or `false`, it can always be simplified into a combination of `&&`s and `||`s.

This is actually a particularly easy one because:
```js
choice.name === 'both' ? true : false
```
is exactly the same as
```js
choice.name === 'both'
```